code/datums/spells/ethereal_jaunt.dm

```diff
@@ -95,6 +95,7 @@
 	var/canmove = 1
 	density = 0
 	anchored = 1
+	invisibility = INVISIBILITY_OBSERVER

```

Comment:
I based on on a PR i saw SOMEWHERE i thouht was related to it....Can't find now...Will change back, Sorry @tigercat2000